File-Formats/Presentation/Getting-Started.md

```diff
@@ -549,6 +549,20 @@ Imports Syncfusion.Pdf
 Imports Syncfusion.PresentationToPdfConverter
 {% endhighlight %}
 
+{% highlight c# tabtitle="ASP.NET Core" %}
+using Syncfusion.Presentation;
+using Syncfusion.OfficeChartToImageConverter;
+using Syncfusion.Pdf;
+using Syncfusion.PresentationRenderer;
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+using Syncfusion.Presentation;
+using Syncfusion.OfficeChartToImageConverter;
```

Comment:
Remove OfficeChartToImageConverter and add System.IO

The namespace is similar to PPTX to PDF page.

Reply:
@Suriya-Balamurugan -Modified